src/main/java/me/dinowernli/grpc/polyglot/server/HelloServiceImpl.java

```diff
@@ -31,4 +32,26 @@ public void sayHelloStream(HelloRequest request, StreamObserver<HelloResponse> r
     }
     responseStream.onCompleted();
   }
+
+  @Override
+  public StreamObserver<HelloRequest> sayHelloBidi(final StreamObserver<HelloResponse> responseStream) {
+    return new StreamObserver<HelloRequest>() {
+      @Override
+      public void onNext(HelloRequest request) {
+        responseStream.onNext(HelloResponse.newBuilder()
+                                 .setMessage("Hello, " + request.getRecipient())
```

Comment:
Please fix the indentation, soo responseStream.onNext and the following lines above.